blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+# Running Graph Analytics on Large Scale Graphs Effortlessly with NVIDIA and Memgraph 
+
+## Introduction
+Since **MAGE's** latest release, you can now run **GPU** powered graph analytics from **Memgraph** in seconds. Loading the whole dataset containing 1M+ edges and getting the first graph analytics results shouldn't take more than 15 seconds :fire: On top of it all, you can do it in **Python**. Let me show you how.
+
+In this tutorial you will learn:
+* how to effortlessly import data inside Memgraph, all while using **Python**
+* how to run analytics on graphs and get results fast 
+* how to run analytics on **NVIDIA GPU** from Memgraph 
+
+What made all of it possible is the integration and collaboration of **Nvidia cuGraph** and **Memgraph** teams :heart:.
+
+In this tutorial I will show you how to use  **PageRank** graph analysis and **Louvain** community detection on a **Facebook dataset** containing 1.3 million edges. After you follow this tutorial, you will know how to run  any of the following graph analytics algorithms with **Python** on your dataset:  
```

Comment:
If you are going to use nodes, then use relationships instead of edges.

There was a discussion that terms have pairs... so vertex and edge are mathematical terms, and relationship and node are graphDB terms... so you either use one or the other... when we were deciding should we use the mathematical or graph DB, we decided that nodes and relationship are more user friendly, as vertices(vertex) is a handful of a term... we started the discussion because people like to mix everything (all four terms) in the same piece of text...

Reply:
Okay makes sense.